### PR TITLE
Run codespell.

### PR DESCRIPTION
**Pull request by drwells**
Part of #10072.

### CODE REVIEW

File: source/multigrid/mg_transfer_component.cc [hunk 1]
```diff
@@ -531,16 +531,16 @@ MGTransferComponentBase::build(const DoFHandler<dim, spacedim> &mg_dof)
                   for (types::global_dof_index i = 0; i < n_dofs; ++i)
                     {
                       SparseMatrix<double>::iterator
-                        anfang = prolongation_matrices[level]
```
Comment (masterleinad) [inline source/multigrid/mg_transfer_component.cc:534]: I personally think, we should use more german!

Comment (drwells) [reply source/multigrid/mg_transfer_component.cc:534]: I could be convinced to leave it :)

Comment (bangerth) [reply source/multigrid/mg_transfer_component.cc:534]: Ha, that's a good one :-)

Comment (bangerth) [reply source/multigrid/mg_transfer_component.cc:534]: I bet that dates back to 1999 when @guidokanschat wrote these lines!

**Pull request merged**